app/src/src/screens/likes/LikesScreen.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useRef, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import Header from '../../components/views/Header';
+import FeedContainer from '../../components/containers/FeedContainer';
+import CreateItemModal from '../../components/modals/CreateItemModal';
+import { CommentsEndpoint } from '../../utils/endpoints';
+import CommentView from '../../components/views/CommentView';
+import UsersList from '../../components/common/UsersList';
+
+const endpoints = [CommentsEndpoint];
+const itemViews = [(item) => <CommentView {...item} />];
```

Comment:
Why do these say comments?

Reply:
Most of the functionality is taken from the way the comments screen is rendered. As we do not have a likes endpoint at the moment, the PR was for you and Trent to make initial suggestions on how we can implement the screen, referring back to how comments are rendered.